README.md

```diff
@@ -55,18 +55,20 @@ In the "Open Project" dialog you might select the option "Open Required Projects
 
 Work on Kata exercises
 ----------------------
-There are two separate katas under different directories (Company Kata and Pet Kata).
+There are three separate katas under different directories (Pet Kata, Company Kata, and Candy Kata).
 
 ```
-company-kata/src/tests
 pet-kata/src/tests
+company-kata/src/tests
+candy-kata/src/tests
 ```
 
 To get started, you can refer to slides for the [Instruction and Pet Kata](http://eclipse.github.io/eclipse-collections-kata/) to learn how to set-up Kata, basic features of Eclipse Collections corresponding to each Pet Kata exercise and then solutions. 
+Check out the [pet kata solutions module tests](https://github.com/eclipse/eclipse-collections-kata/tree/master/pet-kata-solutions/src/test/java/org/eclipse/collections/petkata) for your reference.
 
 To learn wider range of functionalities, slides for [Company Kata](http://eclipse.github.io/eclipse-collections-kata/company-kata) are now available online as well. 
 
-Check out the [pet kata solutions module tests](https://github.com/eclipse/eclipse-collections-kata/tree/master/pet-kata-solutions/src/test/java/org/eclipse/collections/petkata) for your reference.
+To learn more about the Bag data structure, take a look at the [Candy Kata](candy-kata)
```

Comment:
Please add a period at the end of sentence to be consistent with the rest of the paragraphs.